api-auto-registration/troubleshooting.hbs.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting
+
+Here are some handy commands for debugging or troubleshooting the APIDescriptor CR.
+
+1. Find the status of the APIDescriptor CR.
```

Comment:
The `-owide` flag is useful in some cases too. Could you add that as the first tip?